### ISSUE TITLE
开源助手表格测试

### ISSUE THREAD

**Issue by MarioLulab** [PaddlePaddle/Paddle#58281]
报名格式参考：【报名】：2、3，多个任务之间需要使用中文顿号分隔，报名多个连续任务可用横线表示，如 2-5
PR 提交格式：在 PR 的标题中以【PIR API adaptor No.xxx】 开头，注明任务编号

| 序号 |                Python API/所在文件/单测覆盖率                | 认领人/状态/PR号 |
| :--: | :----------------------------------------------------------: | :--------------: |
| 1 |         paddle.abs <br/> `python/paddle/tensor/ops.py`  <br/> 10/15       |                  |
| 2 |     paddle.nn.L1Loss<br />`python/paddle/nn/layer/loss.py`   <br/> 0/8  |                  |
| 3 |     paddle.Tensor.acos<br />`python/paddle/tensor/ops.py`      |                  |
| 4 |     paddle.Tensor.acosh<br />`python/paddle/tensor/ops.py`     |                  |
| 5 | paddle.nn.functional.affine_grid<br />`python/paddle/nn/functional/vision.py` |   |
| 6 |     paddle.Tensor.amax<br />`python/paddle/tensor/math.py`     |                  |

 ## 看板信息 #####

**Comment by Aurelius84** [comment]
【报名】：5、6

**Comment by MarioLulab** [comment]
测试完毕，移步任务发布 issue
